tests/math/test_entropies_math.py

```diff
@@ -224,3 +224,47 @@ def test_size_mismatch(self, state0, state1):
 
         with pytest.raises(qml.QuantumFunctionError, match=msg):
             qml.math.relative_entropy(state0, state1)
+
+
+class TestMinEntropy:
+    """Tests for the min entropy qml.math function"""
+
+    base = [None, 2]
+    check_states = [True, False]
+
+    @pytest.mark.parametrize("interface", ["autograd", "jax", "tensorflow", "torch"])
+    @pytest.mark.parametrize(
+        "state, expected",
+        [
+            ([1, 0], -np.log(1)),
+            ([0, 1], -np.log(1)),
+            ([1, 1] / np.sqrt(2), -np.log(1)),
+            ([1, -1] / np.sqrt(2), -np.log(1)),
+        ],
+    )
+    @pytest.mark.parametrize("base", base)
+    @pytest.mark.parametrize("check_state", check_states)
+    def test_state(self, interface, state, expected, base, check_state):
+        state = qml.math.asarray(state, like=interface)
+        actual = qml.math.min_entropy(state, base=base, check_state=check_state)
+
+        div = 1 if base is None else np.log(base)
+        assert np.allclose(actual, expected / div, rtol=1e-06, atol=1e-07)
+
+    @pytest.mark.parametrize("interface", ["autograd", "jax", "tensorflow", "torch"])
+    @pytest.mark.parametrize(
+        "state, expected",
+        [
+            ([[1, 0], [0, 0]], -np.log(1)),
+            ([[0, 0], [0, 1]], -np.log(1)),
+        ],
+    )
```

Comment:
Can you also include states with non-zero entropy? They shouldn't be too hard to determine by hand.

Reply:
Sure